server/core/storage.go

```diff
@@ -237,18 +237,13 @@ func (s *Storage) DeleteRule(ruleKey string) error {
 
 // LoadRules loads placement rules from storage.
 func (s *Storage) LoadRules(f func(k, v string)) (bool, error) {
-	// Range is ['rule/\x00', 'rule0'). 'rule0' is the upper bound of all rules because '0' is next char of '/' in
-	// ascii order.
-	nextKey := path.Join(rulesPath, "\x00")
-	endKey := rulesPath + "0"
+	nextKey := rulesPath + "/"
+	endKey := clientv3.GetPrefixRangeEnd(nextKey)
 	for {
 		keys, values, err := s.LoadRange(nextKey, endKey, minKVRangeLimit)
 		if err != nil {
 			return false, err
 		}
-		if len(keys) == 0 {
-			return false, nil
```

Comment:
It seems that the first return value is omitted?

Reply:
yes, seems not very useful. I think we can remove it.